packages/jadmin/jadmin.js

```diff
@@ -330,5 +308,58 @@ yargs
       await db.pool.end()
     },
   })
+  .command({
+    command: 'assign-badge <userId> <badgeName>',
+    describe: 'Assign a badge to a user',
+    builder: (yargs) => {
+      yargs.option('force', {
+        describe: 'Bypass interactive safety checks',
+        type: 'boolean',
+      })
+      yargs.option('database-url', {
+        describe: 'The URL to connect to the database',
+        type: 'string',
+      })
+    },
+    handler: async (args) => {
+      const db = await getDb(args)
+      const userId = parseInt(args.userId)
+
+      const user = await db.get`
+        SELECT *
+        FROM "User"
+        WHERE id = ${userId}
+      `
+
+      if (!user) {
+        console.error(`No User with ID ${userId} found`)
+        process.exit(1)
+      }
+
+      const badgeOptionsResponse = (
+        await db.get`
+        SELECT enum_range(NULL::"BadgeType");
+      `
+      ).enum_range
+      const badgeOptions = badgeOptionsResponse.substr(1, badgeOptionsResponse.length - 2)
+      const badgeOptionsArray = badgeOptions.split(',')
+      console.log('badgeName', args.badgeName, typeof args.badgeName)
+      console.log('badgeOptionsResponse', badgeOptionsResponse)
+      console.log('badgeOptions', badgeOptions)
+      console.log('badgeOptionsArray', badgeOptionsArray)
+      if (badgeOptionsArray.includes(args.badgeName)) {
+        await db.query`
+          INSERT INTO "UserBadge"
+          VALUES (DEFAULT, ${args.badgeName}, DEFAULT, ${userId});
+        `
```

Comment:
Try this:

```suggestion
        await db.query`
          INSERT INTO "UserBadge"
          ("type", "userId")
          VALUES
          (${args.badgeName}, ${userId});
        `
```

When you do don't name the column names (`("type", "userId")`) then the assumption is that the VALUES sequence is provided in the order they were listed in the table's CREATE statement. In prod that seems to be createdAt, id, type, userId and based on this I think your local data got id, type, createdAt, userId. From prod:

```
journaly_db=> SELECT * FROM "UserBadge" LIMIT 1;
        createdAt        | id |    type    | userId 
-------------------------+----+------------+--------
 2020-11-09 21:00:22.936 |  1 | ALPHA_USER |     29
 ```